packages/webstudio-sdk/src/css/index.ts

```diff
@@ -0,0 +1,8 @@
+export * from "./properties";
+export * from "./units";
+export * from "./types";
+export * from "./get-browser-style";
+export * from "./categories";
+export * from "./keyword-values";
+export * from "./breakpoints";
+export * from "./schema";
```

Comment:
What parts of this already exist in designer and can be re-used? @kof ?

Reply:
designer uses it from sdk,  we will have to split those in different packages and different packages will come with their own schema